assets/js/base/components/product-grid/container.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import ProductGrid from './index';
+
+class ProductGridContainer extends Component {
+	state = {
+		currentPage: 1,
+		orderValue: null,
```

Comment:
Nitpicking and not sure if this is consistant across the codebase, but I think we should use 'sort' over 'order' where possible simply because 'order' can be confused with orders, whereas sorting is explicit. I realise this doesn't match up with the API however.

Reply:
Yes, make sense. How does it sound handling this in a follow-up? `orderValue` is used also in some _Reviews_ blocks, so I think it's better to deal with it separately from this PR.

Reply:
Sounds good to me.